### PR TITLE
PLAT-143809 Added UI tests for Alert.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Alert had no UI tests.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Created UI tests for Alert.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-143809

### CODE REVIEW

File: tests/ui/specs/Alert/Alert-specs.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+			expect(alertCommon.buttonFullscreen.isFocused()).to.be.true();
+		});
+
+		it('should spot the cancel button', () => {
+			Page.spotlightSelect();
+			Page.spotlightRight();
+
+			expect(components.alertFullscreen.buttonCancel.isFocused()).to.be.true();
+		});
+		it('should close the fullscreen alert using the ok button', () => {
```
Comment (stanca-pop-lgp) [inline tests/ui/specs/Alert/Alert-specs.js:22]: Please leave an empty line between tests.

File: tests/ui/specs/Alert/Alert-specs.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+		});
+		it('should close the overlay alert using the close button', () => {
+			Page.spotlightRight();
+			Page.spotlightSelect();
+			Page.spotlightDown();
+			Page.spotlightSelect();
+
+			expect(alertCommon.buttonOverlay.isFocused()).to.be.true();
+		});
+	});
```
Comment (stanca-pop-lgp) [inline tests/ui/specs/Alert/Alert-specs.js:43]: Maybe you could also add a test for when pressing Esc key (Page.backKey();) and one when clicking outside the alert popup. You can take https://github.com/enactjs/sandstone/pull/945/files as an example.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

File: tests/ui/specs/Alert/Alert-specs.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
+const Page = require('./AlertPage');
+const {expectClosed, expectOpen, validateTitle} = require('./Alert-utils.js');
```
Comment (dongsuwon) [inline tests/ui/specs/Alert/Alert-specs.js:2]: Need an empty line below here.

File: tests/ui/apps/Alert/Alert-View.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import Alert from '../../../../Alert';
+import Button from '../../../../Button';
+import ThemeDecorator from '../../../../ThemeDecorator';
+import spotlight from '@enact/spotlight';
+import {Component} from 'react';
+import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator';
```
Comment (dongsuwon) [inline tests/ui/apps/Alert/Alert-View.js:6]:
```suggestion
	import spotlight from '@enact/spotlight';
	import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator';
	import {Component} from 'react';
	
	import Alert from '../../../../Alert';
	import Button from '../../../../Button';
	import ThemeDecorator from '../../../../ThemeDecorator';
`````
Please sort by package name.

File: tests/ui/specs/Alert/Alert-specs.js (hunk 3 above)
Comment (alexandrumorariu) [reply tests/ui/specs/Alert/Alert-specs.js:2]: Done

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**